src/DotNetWorker/GrpcWorker.cs

```diff
@@ -53,7 +115,6 @@ private Task ProcessRequestCoreAsync(StreamingMessage request)
                 MsgType.WorkerInitRequest => WorkerInitRequestHandlerAsync(request),
                 MsgType.FunctionLoadRequest => FunctionLoadRequestHandlerAsync(request),
                 MsgType.InvocationRequest => InvocationRequestHandlerAsync(request),
-                MsgType.FunctionEnvironmentReloadRequest => FunctionEnvironmentReloadRequestHandlerAsync(request),
```

Comment:
I think we still want `FunctionEnvironmentReloadRequest` and its handler?

Reply:
We weren't doing anything with it today. I believe @pragnagopa said that we didn't need to support it? I can re-add it though...

Reply:
I added it back. Didn't want to get too crazy.